src/components/Tooltip/index.tsx

```diff
@@ -9,12 +9,16 @@ const TooltipContainer = styled.div`
   font-weight: 400;
 `
 
+const InputContainer = styled.div`
+  width: 388px;
```

Comment:
due to this fixed width its breaking in mobile.
![2021-09-24_21-55](https://user-images.githubusercontent.com/6604146/134709534-80c9232a-2dce-48bd-a057-533f8b3eac09.png)